apps/prairielearn/src/lib/externalGradingSocket.ts

```diff
@@ -92,7 +92,6 @@ export function connection(socket: Socket) {
       user_id: msg.user_id,
       urlPrefix: msg.url_prefix,
       questionContext: msg.question_context,
-      csrfToken: msg.csrf_token,
```

Comment:
You can remove the csrf_token from the `ensureProps` call a few lines before this point. The socket may still be sending them, but you don't need to enforce it is there.